tests/test_export.py

```diff
@@ -0,0 +1,50 @@
+import os
+import tempfile
+
+import numpy as np
+import pytest
+
+import gustaf as gus
+
+all_grids = (
+    ("volumes_hexa", "mfem_hexahedra_3d.mesh"),
```

Comment:
any chance you could add 2D?

Reply:
I added it. It's a bit ugly, but I'm not familiar with pytest fixtures. @clemens-fricke do you have an idea of how I could make it better? 

Specifically this part for example:
```Python
@pytest.fixture
def faces_quad(vertices_3d, quad_connec):
    return gus.Faces(vertices_3d, quad_connec)


@pytest.fixture
def faces_quad_2d(vertices_2d, quad_connec_2d):
    return gus.Faces(vertices_2d, quad_connec_2d)
```

Is there maybe a way to include the dimensionality as a parameter? Or should we leave it like this?